server/auth/config/passport.js

```diff
@@ -0,0 +1,43 @@
+const LocalStrategy = require('passport-local').Strategy
+// need a User model in db or something of equivalence (mock import)
+const { User } = require('../models')
+
+module.exports = function (passport) {
+  passport.use(
+    new LocalStrategy(
+      {
+        // by default, local strategy uses username, we will override with email
+        usernameField: 'email'
+      },
+      async (email, password, done) => {
+        try {
+          const user = await User.findOne({
```

Comment:
We will have to update this config to use `Admin`s. For password validation, check out `server/lib/encrypt` after you rebase this. That will have the methods you need.

Reply:
Now that the `Admin` model exists and has been implemented, the mock User model has been removed and config updated. Password validation now being handled by the code in `server/lib/encrypt`